scripts/ci/dockerhub/centos7-gcc8-clang6.0/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM centos:centos7
+LABEL William F Godoy williamfgc@yahoo.com
+RUN yum update -y &&\
+    yum upgrade -y &&\
+    yum install centos-release-scl -y &&\
+    yum install devtoolset-8-gcc devtoolset-8-gcc-c++ llvm-toolset-6.0 -y &&\
```

Comment:
@mabrahamdevops thanks! We might need to add the following before being able to install `llvm-toolset-6.0` :

```
echo "[SCL-core]
name=Springdale SCL Base $releasever - $basearch
mirrorlist=http://springdale.princeton.edu/data/springdale/SCL/$releasever/$basearch/mirrorlist
#baseurl=http://springdale.princeton.edu/data/springdale/SCL/$releasever/$basearch
gpgcheck=0" >> /etc/yum.repos.d/CentOS-Base.repo
```

Reply:
I just reviewed the video.  I think I distracted you with a question from explaining the above code snippet.  I just want to confirm that this should go in the new Centos7-gcc8clang6.0 > Dockerfile to pull the image?

Reply:
Yup it should go! The `llvm-toolset-6.0` package which has `clang 6.0` is provided by a well-known third party (Springdale), this is just adding it to the lists of known repos. Once this PR is merged I can create and push the docker container to DockerHub since it's still under my account. Thanks a lot!